From 1604d2a25ab6bb838ab38fa8de26688882e9f32e Mon Sep 17 00:00:00 2001
From: Aymen Sghaier <aymen.sghaier@nxp.com>
Date: Tue, 19 Jun 2018 11:29:00 +0200
Subject: [PATCH 4200/5242] MLK-17227: crypto: caam: Update CAAM driver
 verbosity while RNG init

commit  bc328d847a575082472d7aebbfe0610c5adde32a from
https://source.codeaurora.org/external/imx/linux-imx.git

  Move to debug level rather than error level the RNG init traces while
  increasing entropy delays.

Signed-off-by: Aymen Sghaier <aymen.sghaier@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/crypto/caam/inst_rng.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/crypto/caam/inst_rng.c b/drivers/crypto/caam/inst_rng.c
index ae1846d..7ab379d 100644
--- a/drivers/crypto/caam/inst_rng.c
+++ b/drivers/crypto/caam/inst_rng.c
@@ -156,8 +156,8 @@ static int instantiate_rng(int state_handle_mask, int gen_sk)
 		/* Try to run it through JR */
 		ret = run_descriptor_jr(desc, sh_idx);
 		if (ret)
-			pr_err("Failed to run desc  RNG4 SH%d status (0x%x)\n",
-			       sh_idx, ret);
+			pr_debug("Failed to run desc  RNG4 SH%d status (0x%x)\n",
+				 sh_idx, ret);
 		/* Clear the contents before recreating the descriptor */
 		memset(desc, 0x00, CAAM_CMD_SZ * 7);
 	}
@@ -204,8 +204,8 @@ int deinstantiate_rng(int state_handle_mask)
 			/* Try to run it through JR */
 			ret = run_descriptor_jr(desc, sh_idx);
 			if (ret)
-				pr_err("Failed to run desc to deinstantiate RNG4 SH%d\n",
-					  sh_idx);
+				pr_debug("Failed to run desc to deinstantiate RNG4 SH%d\n",
+					 sh_idx);
 		}
 	}
 
-- 
1.7.9.5

